lldb/test/Shell/SymbolFile/DWARF/static_scope.s

```diff
@@ -0,0 +1,312 @@
+# Test that the DWARF parser assigns the right scope to the
+# variable `b`, which is `local` and not `static`.
+
+# REQUIRES: x86
+# UNSUPPORTED: lldb-repro
```

Comment:
What's the deal here (could you leave a brief/1liner comment about why lldb-repro isn't supported)?

Reply:
This is a cherry-pick from upstream. I guess you might want to comment there.
The only relevant swift bit is the one I outlined before [the check removed].